lib/python/vdm/server/HTTPListener.py

```diff
@@ -173,11 +183,11 @@ def map_deployment(request, database_id):
     if 'cluster' in request.json and 'kfactor' in request.json['cluster']:
         deployment[0]['cluster']['kfactor'] = request.json['cluster']['kfactor']
 
-    if 'adminMode' in request.json and 'adminstartup' in request.json['adminMode']:
-        deployment[0]['adminMode']['adminstartup'] = request.json['adminMode']['adminstartup']
+    if 'admin-mode' in request.json and 'adminstartup' in request.json['admin-mode']:
```

Comment:
this translation layer is exceedingly fragile. it must be refactored for the next release. how in heaven's sake can it handle two version of VoltDB server that  support differing deployment file schemas